src/explore-education-statistics-admin/src/pages/release/datablocks/components/DataBlockPageTabs.tsx

```diff
@@ -377,7 +377,6 @@ const DataBlockPageTabs = ({
           <TabsSection title="Data source" id="dataBlockTabs-dataSource">
             {!isLoading && tableState && (
               <DataBlockSourceWizard
-                key={saveNumber}
```

Comment:
We still use `saveNumber` for the chart tab. Do we still need this?

Reply:
It's used to prevent an error when you have a chart and change the filters etc. Though I have just found another bug while testing that.... I'll raise a separate ticket for it.

Reply:
Cool!